CONTRIBUTING.md

```diff
@@ -101,9 +101,13 @@ For further detail on integrating third party services, fonts, embeds, etc. see
 
 ## Contributor License Agreement
 
-The AMP Project hosted at GitHub requires all contributors to sign a Contributor License Agreement ([individual](https://developers.google.com/open-source/cla/individual) or [corporation](https://developers.google.com/open-source/cla/corporate)) in order to protect contributors, users and Google in issues of intellectual property.
+The AMP Project hosted at GitHub requires all contributors to sign a Contributor License Agreement in order to protect contributors, users and Google in issues of intellectual property.
 
-When you create a Pull Request a check will be run to ensure that you have signed the CLA.  Make sure that you sign the CLA with the same email address you associate with your commits (likely via the `user.email` Git config as described on GitHub's [Set up Git](https://help.github.com/articles/set-up-git/) page).
+**We recommend signing the CLA *before* you send a pull request to avoid problems.  Make sure that you sign the CLA with the same email address you associate with your commits (likely via the `user.email` Git config as described on GitHub's [Set up Git](https://help.github.com/articles/set-up-git/) page).**
```

Comment:
Do you think making it a blocking step to submission is worth it? Might be better to lower initial barriers, since it is fine if this only happens eventually. Often corp CLAs can take a while.